migration/entity/src/entities/passport.rs

```diff
@@ -0,0 +1,50 @@
+//! `SeaORM` Entity, @generated by sea-orm-codegen 1.0.0-rc.5
+
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Serialize, Deserialize)]
+#[sea_orm(table_name = "passport")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub owner_id: i32,
+    pub version: i32,
+    pub surname: String,
+    pub name: String,
+    pub date_of_birth: Date,
+    pub date_of_issue: Date,
+    pub place_of_origin: String,
+    pub secret: String,
+    pub activated: bool,
+    #[sea_orm(unique)]
+    pub ceremony_time: DateTime,
+}
+
+#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
+pub enum Relation {
+    #[sea_orm(
+        belongs_to = "Entity",
+        from = "Column::CeremonyTime",
+        to = "Column::CeremonyTime",
+        on_update = "Cascade",
+        on_delete = "Cascade"
+    )]
+    SelfRef,
```

Comment:
This is incorrect, it's self referential. Your `to` is wrong.

Reply:
Whoops... Fixed